eng/scripts/Approve-Eng-Common-Sync-PRs.ps1

```diff
@@ -59,6 +59,6 @@ foreach ($repo in $repos)
   }
 
   if ($prstate.mergeStateStatus -ne "CLEAN") {
-    Write-Host "****PR is not mergeable [$($prstate.mergeStateStatus)] and may need to be manually merged"
+    Write-Host "****PR $($prstate.number) is not mergeable [state: $($prstate.mergeStateStatus)] and may need to be manually merged"
```

Comment:
adding it to avoid confusion of which line/PR this output belongs to and to understand the meaning of the output

Reply:
I would suggest just logging the url that way folks can copy and past it as well to look at it closer.

Reply:
url added